pom.xml

```diff
@@ -69,6 +70,17 @@
       <artifactId>cbioportal</artifactId>
       <version>b6b7b3f8332b94f01adc9e9263af8f2fe2b4e09a</version>
     </dependency>
+    <!-- api client -->
```

Comment:
These api client dependencies only work locally for now, we should replace this with the proper jitpack version once we merge https://github.com/genome-nexus/genome-nexus-java-api-client/pull/1

Reply:
@inodb replaced local dependencies with the jitpack ones, I think this PR is ready to merge.